README.md

```diff
@@ -1,25 +1,33 @@
-|-------------------    |-------------------------    |------------------------    |-----------------------    |
-| URL                   | epr-snapshot.elastic.co     | epr-staging.elastic.co     | epr.elastic.co            |
-| Add package           | Commit                      | Commit                     | PR                        |
-| Version overwrite     | yes                         | if needed                  | no                        |
-| Stack Version         | *-SNAPSHOT                  | *-SNAPSHOT                 | Released (candidates)     |
-| Registry Version      | fixed version                | Stable release             | Stable release            |
-| Branch                | snapshot                    | staging                    | production                |
-| Packages              | snapshot+staging+prod       | staging+production         | production                |
-| Release               | Manual                      | Manual                  | Manual                 |
-| Docker image          | snapshot                    | staging                    | production                |
+|                         | Snapshot                    | Staging                    | Production                |
+|-------------------      |-------------------------    |------------------------    |-----------------------    |
+| URL                     |  [epr-snapshot.elastic.co](https://epr-snapshot.elastic.co/search) | [epr-staging.elastic.co](https://epr-staging.elastic.co/search) | [epr.elastic.co](https://epr.elastic.co/search) |
+| How to Add package      | Commit to elastic/integrations* | PR with elastic-package tool | PR with elastic-package tool |
+| Allow Version overwrite?| yes**                        | if needed                  | no                        |
+| Allow Version removal?  | yes                         | special exceptions only    | only version increments   |
+| Stack Ver vs Storage Ver| [master branch](https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/server/services/epm/registry/registry_url.ts#L25) | none set to staging | all non-master branches*** |
+| Registry Version        | fixed dev or latest stable release | Stable release      | Stable release            |
+| Branch                  | snapshot                    | staging                    | production                |
+| Packages                | snapshot+staging+prod       | staging+production         | production                |
+| Release                 | Manual                      | Manual                     | Manual                    |
+| Docker image            | snapshot                    | staging                    | production                |
+
+* https://github.com/elastic/integrations is the development home of most Integrations source, tho not all, promotion process to storage repos is discussed below
```

Comment:
I think you'll want to put the `*` in backticks or else it'll be rendered as a bullet.

```suggestion
`*` https://github.com/elastic/integrations is the development home of most packages, though not all. Promotion process to package storage repository is discussed below.
```